### ISSUE TITLE
project dead?

### ISSUE THREAD

**Issue by nerdoc** [TheCulliganMan/tetra#71]
Hi @TheCulliganMan,

do you still use this project, is it in active development? The docs don't even say anything about what it does...

I'm just asking because there is another open source project named [Tetra framework](tetraframework.com) which seems to be (and become) quite a cool thing. And if you don't need this name - especially on [PyPi](https://pypi.org/project/tetra/), it would be very kind of you to hand it to @samwillis - If not, I can understand completely - I'm just asking :smile:

This would help e.g. not importing the wrong library - both are imported using `from tetra import ...` - and IDEs like PyCharm try to install `tetra` then, and not `tetraframework`.

Cheers,
Christian

**Comment by TheCulliganMan** [comment]
Thanks for reaching out.  I haven't used this one in a while and would be glad to transfer the PyPi handle over to @samwillis if he gets in contact.

**Comment by nerdoc** [comment]
Thanks, that's really awesome. It's so nice to see that there are still a great bunch of people around that try to help each other.
@samwillis, could you chime in here maybe and have a talk with @TheCulliganMan about the details?

**Comment by nerdoc** [comment]
Hey @TheCulliganMan, sorry to have such a long break before hearing from me again. 
I had not reached @samwillis for a long time now as he had been very busy with other work, and he told me he most probably won't have time to work further on his [tetraframework](https://github.com/samwillis/tetra). I offered him to take over ownership or maintenance of his tetra framework a few days ago - as I would definitely like to develop it further. I waiting for his response ATM.

But I wanted to ask you, if your offer to hand over the pypi handle is still valid.

If yes, I would take it with great respect - assumed that @samwillis decides to hand over his repo to me - I would like to still wait for that to happen, to not overhastily do things. Just checking what you say about it.
Greetings,
Christian

**Comment by TheCulliganMan** [comment]
Yes, I'm traveling at the moment, but would be glad to do this next week.

**Comment by samwillis** [comment]
Hey everyone 👋 

Life got busy and I sadly didn't have time to work on this. I still very much believe in the concept, Django is crying out for a great full stack component framework.

I'm so thankful to @nerdoc for asking to pick up where I left it, I've added him as a collaborator to the repo.

I'm going to see if I can answer questions and offer guidance as he takes it forward.

**Comment by nerdoc** [comment]
Hello Ryan, hope you had a pleasant trip! If you find some time to do the transition, this would be really cool! I'm afraid I can't help you here...

**Comment by nerdoc** [comment]
Hi @TheCulliganMan again - may I ping you again and ask for the transfer - on my side everything is on track :smile: - thanks!
